### DISCUSSION TITLE
WSL installation error

### DISCUSSION THREAD

**Discussion by voiderd-sub** [ANLAB-KAIST/KENSv3#48] [Project Q&A 4. Etc] [answered]
![image](https://user-images.githubusercontent.com/102510197/160348866-7ee4383b-2e36-4d1c-93b9-5e3d877a368b.png)

이미지와 같이 에러가 발생해 과제를 진행하지 못하고 있습니다...

검색을 통해 시도해본 방법은 다음과 같습니다. (참고 링크 : https://ko.purelysandy.com/wslregisterdistribution-failed-with-error-0x80370114)
1) Windows 기능 켜기/끄기에서 Hyper-V 및 Linux용 Windows 하위 시스템 활성화
2) 관리자 권한으로 연 PowerShell에 Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux 입력
3) Windows 보안 탭의 Exploit Protection->프로그램 설정 탭에서 vmcompute.exe에서 CFG (Code Flow Guard) 부분 체크 해제하고 PowerShell에 net start vcompute 입력 (이때 서비스 이름이 잘못되었다고 출력됨)
4) LxssManager 서비스 재시작
5) Ubuntu 복구 및 재설치
6) Debian 등 다른 distribution 사용 (모두 ubuntu와 같은 출력이 나타남)

어떻게 하는 게 좋을까요?

**Comment by Pusnow** [comment, marked as answer]
Unfortunatly, I also don't know... I'm not a Windows expert. You can contact [Microsoft community](https://answers.microsoft.com/).

Or you can use Virtualbox.
